punishments.js

```diff
@@ -1081,6 +1088,18 @@ Punishments.monitorRoomPunishments = function (user) {
 			return `<<${room}>> (${punishDesc})`;
 		}).join(', ');
 
-		Monitor.log(`[PunishmentMonitor] ${user.name} currently has punishments in ${punishments.length} rooms: ${punishmentText}`);
+		if (Config.enablepunishmentautolock && roombans.length && (roombans.length + blacklists.length) >= 3) {
```

Comment:
`enable` is kinda superfluous in a Config setting.

`Config.punishmentautolock`